docs/docsite/rst/network/user_guide/network_best_practices_2.5.rst

```diff
@@ -107,6 +89,21 @@ Ansible vault for password encryption
 
 The "Vault" feature of Ansible allows you to keep sensitive data such as passwords or keys in encrypted files, rather than as plain text in your playbooks or roles. These vault files can then be distributed or placed in source control. See :ref:`playbooks_vault` for more information.
 
+Here's what it would look like if you specified your SSH passwords (encrypted with Ansible Vault) among your variables:
+
+.. code-block:: yaml
+
+   ansible_connection: network_cli
+   ansible_network_os: vyos
```

Comment:
```suggestion
   ansible_network_os: vyos.vyos.vyos
```

Reply:
I realize the rest of the page doesn't have FQCN, but this will be two less I have to update later :-)